.github/workflows/build_docs.yml

```diff
@@ -33,6 +37,7 @@ jobs:
         with:
           # Check out to  '/home/runner/work/docs/docs/docs'
           path: docs
+          ref: ${{ inputs.pr_ref || github.ref }}
```

Comment:
Here make sure to be able to use the branch info passed in from the triggering job or just the regular dispatch.

Reply:
adding branch to `pr_ref` name would improve readability and clarity (or renaming to `pr_branch`)

Reply:
If i recall `ref` is the GitHub terminology. I will try to circle around to this and see if it can be changed.

Reply:
Yeah, as noted in the other spot, `ref` is actually git terminology and it is also used by GitHub. So we either use the `github.ref` if using a workflow dispatch or the `pr...ref` which we have stored in the `pr_ref` variable. So i lean to leaving this to make it easier for someone searching/reading github/git docs to sort this out?